src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -98,8 +97,10 @@ export class NotebookEditor implements INotebookEditor {
         return getNotebookMetadata(this.document);
     }
     onExecutedCode?: Event<string> | undefined;
-    public getContent() {
-        return this.serializer.serializeNotebookDocument(this.document);
+    public async getContent(): Promise<string> {
+        await workspace.saveAll(false);
```

Comment:
Shouldn't we just save the active notebook editor instead of everything. Else user will get provided to save other temporary untiled files as well, which would be undesirable (eg generally I have a few untitled files)

Reply:
What is the export command supposed to do? I run it and nothing happens. Also I think the new method won't work if you export a notebook with unsaved changes. 

I don't want to share code with npm. Does it make sense for vscode to own the export commands?

Reply:
Export runs `nbconvert` which is a Python module.